vuepress/docs/.vuepress/dist/docs/404.html

```diff
@@ -3,17 +3,17 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1">
-    <title>Getting Started Guide</title>
+    <title>VuePress</title>
```

Comment:
This doesn't look right to me.

Reply:
@es-entando seemed to remove the title, which originally was "Getting Started Guide". Vuepress by default generates that title when left empty.